### PR TITLE
Added: Modal, Grid

### PR DESCRIPTION
**Pull request by NeerajVodala**
Added components:
- Modal
- grid

Deploy Preview: 
- https://deploy-preview-9--cluach.netlify.app/docs/components/modal.html
- https://deploy-preview-9--cluach.netlify.app/docs/components/grid.html

Image for reference: 

modal
grid

### CODE REVIEW

File: docs/components/modal.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+&lt;div class="modal shadow-1 br-s"&gt;
+
+    &lt;div class="flex-row align-center justify-between"&gt;
+        &lt;h3 class="modal-heading"&gt;Modal Header&lt;/h3&gt;
+        &lt;i class="fas fa-times fa-lg"&gt;&lt;/i&gt;
+    &lt;/div&gt;
+
+    &lt;p class="modal-text"&gt;modal text.&lt;/p&gt;
+
+    &lt;div class="flex-row align-center justify-end gp-m"&gt;
```
Comment (LordUttam) [inline docs/components/modal.html:140]: Run prettier here as well.

File: docs/components/grid.html [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+
+                <div class="code-snippet br-s">
+                    <pre>
+                        <code class="html">
+
+&lt;div class="grid col-2 grid-gap-05"&gt;
+
+    &lt;div&gt;Column 1&lt;/div&gt;
+
+    &lt;div&gt;Column 2&lt;/div&gt;
```
Comment (LordUttam) [inline docs/components/grid.html:131]: Use prettier to reformat these.

**Review by LordUttam — APPROVED** [review]
Reformat the &lt; &gt; and maybe try renaming the class `.html` to something else which is a bit more descriptive ex: `.html-code`. Rest looks good to me.

**Review by zabihshaik — APPROVED** [review]
Everything looks good. Clean UI.

**Pull request merged**